lib/faraday/options.rb

```diff
@@ -252,7 +252,8 @@ def new_builder(block)
   end
 
   class Env < Options.new(:method, :body, :url, :request, :request_headers,
-    :ssl, :parallel_manager, :params, :response, :response_headers, :status)
+    :ssl, :parallel_manager, :params, :response, :response_headers, :status,
+    :reason_phrase)
```

Comment:
@mislav, I think that `status_text` would be a little more intuitive, as "Status Text" seems to be a common colloquial term. (E.g. https://developer.mozilla.org/en-US/docs/Web/HTTP/Response_codes).

However, "Reason Phrase" is the proper technical term per http://www.w3.org/Protocols/rfc2616/rfc2616-sec6.html, so I'd prefer to keep the name `reason_phrase`, if that's alright with you.

What do you think?